Library/Formula/conan.rb

```diff
@@ -0,0 +1,72 @@
+class Conan < Formula
+  desc "This tool is a distributed, open source, package manager for C/C++"
```

Comment:
This can be simplified to just "Distributed, open source, package manager for C/C++"